src/player.js

```diff
@@ -52,12 +55,17 @@ export const createPlayer = (level, image) => {
     render() {
       this.context.save();
       this.context.translate(this.x, this.y);
+      this.context.scale(1, 1);
 
       if (this.fallingToGround) {
         // Rotation is around top left corner, adjust accordingly:
         this.context.translate(0, this.height);
 
         this.context.rotate(-Math.PI / 2);
+      } else if (moveLeft) {
+        this.context.scale(-1, 1); // mirror player
```

Comment:
Scaling this way also moves the player horizontally, since the 0-coordinate is at the left edge of the image. Check out
https://eloquentjavascript.net/17_canvas.html#p_9a1O8aEtUA